From 2df4a29b54ec9aceec2886f1ac5f8beaaa8d5687 Mon Sep 17 00:00:00 2001
From: Lukasz Majczak <lukaszm@marvell.com>
Date: Wed, 16 Oct 2019 10:44:03 +0200
Subject: [PATCH 343/345] MIPS: Octeon: Take all memory into use by default.

Take all memory into use by default, instead of limiting to 512 MB.

Signed-off-by: Aaro Koskinen <aaro.koskinen@nokia.com>
Acked-by: David Daney <david.daney@cavium.com>
Cc: Sivasubramanian Palanisamy <sivasubramanian.palanisamy@nokia.com>
Cc: linux-mips@linux-mips.org
Patchwork: https://patchwork.linux-mips.org/patch/13353/
Signed-off-by: Ralf Baechle <ralf@linux-mips.org>

This would normally be a cherry-pick of 4c0f5a7124aa "MIPS: Octeon: Take
all memory into use by default.", but it is only a one line change that
doesn't apply because the variable name has changed.

Change-Id: I5dc21178ef2652f9a6d42887a0ffc310c6a1dafd
Signed-off-by: Henry Willard <henry.willard@oracle.com>
---
 arch/mips/cavium-octeon/setup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/cavium-octeon/setup.c b/arch/mips/cavium-octeon/setup.c
index 7e4e8f677c72..ac9b6a8f4bdb 100644
--- a/arch/mips/cavium-octeon/setup.c
+++ b/arch/mips/cavium-octeon/setup.c
@@ -73,7 +73,7 @@ const bool octeon_should_swizzle_table[256] = {
 };
 EXPORT_SYMBOL(octeon_should_swizzle_table);
 
-static unsigned long long max_memory = 512ull << 20;
+static unsigned long long max_memory = ULLONG_MAX;
 static unsigned long long reserve_low_mem;
 
 /*
-- 
2.25.1

